pkg/operator/ceph/cluster/controller.go

```diff
@@ -177,7 +177,7 @@ func (c *ClusterController) onK8sNodeAdd(obj interface{}) {
 
 		if valid, _ := k8sutil.ValidNode(*newNode, cluster.Spec.Placement.All()); valid == true {
 			logger.Debugf("Adding %s to cluster %s", newNode.Labels[apis.LabelHostname], cluster.Namespace)
-			err := cluster.createInstance(c.rookImage, cluster.Info.CephVersion)
+			_, err := cluster.createInstance(c.rookImage, cluster.Info.CephVersion, cephv1.ClusterStateCreating, cluster.Info.Name)
```

Comment:
How about if we rename `cephv1.ClusterStateCreating` to `cephv1.ClusterStateInProgress`? This has been on my mind for a while that "Creating" doesn't make sense after the first orchestration is done. "InProgress" sound ok?